internal/tokenutil/jwtmiddleware.go

```diff
@@ -0,0 +1,40 @@
+package tokenutil
+
+import (
+	"net/http"
+	"strings"
+
+	jwt "github.com/golang-jwt/jwt/v4"
+)
+
+var secretKey = []byte("my_secret_key")
```

Comment:
As a good exercise, consider using ENV variable to store your secret key, as it will never ever be stored in code like that in the open. Also, good point will be to not read it from environment every time you need it, think of the way, how you can do it.